lesson5/index.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+
+const Todo = require('./models/todo')
+
+app.use(bodyParser.urlencoded({ extended: true }))
+
+const port = 8000
+app.listen(port, () => {
+  console.log('Listening on port ' + port)
+})
+
+app.get('/',  async (req, res) => {
+  res.redirect('/todos')
+})
+
+app.get('/todos',  async (req, res) => {
+  const todos = await Todo.find()
+  res.send(todos)
+})
+
+app.post('/todos',  async (req, res) => {
+  try {
+    const message = await new Todo().addTodo(req.body)
+    res.send(message)
+  }
+  catch(err) {
+    console.log(err)
+    res.send('Something went wrong!')
+  }
+})
+
+app.put('/todos/:id',  async (req, res) => {
+  try {
+    const message = await new Todo().updateTodo(req.params.id, req.body)
```

Comment:
Looks strange. This method should be static in a current use case or as a preferable way it should have the following API:
```javascript
const todo = Todo.findById(req.params.id);
todo.updateTodo(req.body);
```

Reply:
Agree. Will make it static.

Reply:
const todo = Todo.findById(req.params.id);
todo.updateTodo(req.body);

I don't like methods like findById to be outside of model. I believe that best way is to incapsulate all db methods in one place. So static method looks like more convenient approach in this case.